pkg/handler/configs.go

```diff
@@ -632,7 +666,7 @@ func (h *Handler) RemoveRepostReactionConfig(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errors.New("guild_id is required"), nil))
 		return
 	}
-	if req.Emoji == "" {
+	if req.Emoji == "" || (req.Emoji == "" && req.EmojiStop == "") {
```

Comment:
how `(req.Emoji == "" && req.EmojiStop == "")` happen when the first condition failed (`req.Emoji != ""`)